examples/daal4py/run_examples.py

```diff
@@ -129,14 +131,20 @@ def check_library(rule):
 req_version['decision_forest_classification_default_dense_batch.py'] = (2023, 'P', 1)
 req_version['decision_forest_classification_traverse_batch.py'] = (2023, 'P', 1)
 req_version['decision_forest_regression_hist_batch.py'] = (2021, 'P', 200)
+req_version['basic_statistics_spmd.py'] = (2023, 'P', 1)
+req_version['linear_regression_spmd.py'] = (2023, 'P', 1)
 
 req_device = defaultdict(lambda: [])
+req_device['basic_statistics_spmd.py'] = ["gpu"]
```

Comment:
Are we targeting examples for GPU only? in this case we could run them only on GPU enabled systems, but i would assume that technically we can run this on cpus as well?

Reply:
We are targeting only these exact examples on GPU. https://github.com/intel/scikit-learn-intelex/blob/master/examples/sklearnex/spmd_basic_statistics.py#L43

Reply:
yes for now, but more generic question is that we can run them on CPU device as well for purpose of validation. This might require change in example

Reply:
Yeah, current examples requires GPU devices, so that is why we have required device check here. It is possible run them on CPU as well, if algo supported on distr CPU. But it requires changes in examples and updates for devices checks in this runner.